src/test/java/uk/gov/companieshouse/presenter/account/PresenterAccountApiApplicationTests.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.companieshouse.presenter.account;
+
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class PresenterAccountApiApplicationTests {
```

Comment:
You don't need the this to plural right?
```suggestion
class PresenterAccountApiApplicationTest {
```

Reply:
done.